requirements.txt

```diff
@@ -1,5 +1,6 @@
 fiona
 geopandas
+geoviews
```

Comment:
> Paying attention to the geoviews install challenges is especially important (many users are still new to creating and updating environments, so this presents an extra hurdle for them). It's also the cause of the Travis CI failure - it needs PROJ to install cartopy, which is a geoviews dependency. I'll try to look into how other packages handle this challenge unless you already have a proposed solution.

Installing `PROJ` into Travis CI will be a bit involved. Either we install it using `conda` (see https://docs.conda.io/projects/conda/en/4.6.1/user-guide/tasks/use-conda-with-travis-ci.html#the-travis-yml-file), or via something like `apt` (https://docs.travis-ci.com/user/languages/python/#dependency-management). This PR looks good, but we might have to wait to handle this after release v0.3.3 (#170), and it might be best to add `geoviews` as a dependency in a separate Pull Request to keep things manageable.

Locally, I've been using `conda create --name icepyx --file requirements.txt` (to create a fresh `icepyx` conda environment) to test, rather than `pip install requirements.txt`. Will need to figure out how to translate that to work on `.travis.yml` (read: may need to setup miniconda first).

Reply:
I've added an issue for further discussion on this: #177

Reply:
Ok, I've opened a separate PR at #193 to handle the `geoviews` installation as an optional dependency in Travis CI. Once that is reviewed (feel free to give a go @icetianli) and approved/merged, we should be able to remove this `geoviews` line from requirements.txt and (finally) merge in this PR!

```suggestion
```

Reply:
@icetianli Did you want to give PR #193 a review, or should I plan to do that? I'd like to move forward with merging this PR and I think that's the only holdup.

Reply:
Thanks @weiji14 and @JessicaS11 for putting this together, I'll give it a go today!